localcluster/scripts/pix-demo.sh

```
@@ -0,0 +1,639 @@
+# feature; `PIX_POOL` (default `secp256k1`) says which one this run expects, and the binary is
+# checked against it before the cluster is started. `PIX_POOL=curvy` selects the Baby JubJub pool,
+# which is currently a stub that panics — it exists so the wiring can be exercised end to end.
+#
+# Safe to re-run: a stale chain container or leftover nodes from an interrupted attempt are
+# cleared on the way in, and Ctrl-C tears the cluster down on the way out.
+
+set -uo pipefail
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+API_PORT_BASE=13500
+NODES=(Entry Relay Exit)
```

Shellcheck (0.11.0)

[warning] NODES appears unused. Verify use (or export if used externally).

(SC2034)